Classes/Command/SentryCommandController.php

```diff
@@ -66,16 +71,6 @@ public function testCommand(): void
         $this->outputLine('This command will now throw an exception for testing purposes.');
         $this->outputLine();
 
-        $this->throwException('Some argument');
-    }
-
-    /**
-     * @param string $someArgument
-     * @throws SentryClientTestException
-     */
-    protected function throwException(string $someArgument): void
-    {
-        $previousException = new \RuntimeException('Test "previous" exception thrown by the SentryCommandController', 1614759554);
-        throw new SentryClientTestException('Test exception in SentryCommandController', 1614759519, $previousException);
+        (new ThrowingClass())->throwException(new StringableTestArgument((string)M_PI));
```

Comment:
Why does that need to be in braces? Did you mean this:

```php
new ThrowingClass())->throwException(new StringableTestArgument((string)M_PI);
```

Reply:
It's usd instead of:

```php
$c = new ThrowingClass();
$c->throwException(new StringableTestArgument((string)M_PI));
```

`(new ThrowingClass())` is needed to avoid `Parse error: syntax error, unexpected token "->" in`

Reply:
I see – never stumbled over that, probably because you usually assign the new object to some variable.